trk/src/Screens/ClimbCreate/index.js

```diff
@@ -125,139 +187,169 @@ const ClimbInputData = () => {
   }
 
 
+
```

Comment:
When they press delete.. with a subtlety: Alert pop up, just to make sure it's not a miss-input by the user.